_data/operations.yml

```diff
@@ -362,3 +362,12 @@ members:
   name: Shawn Tan
   orcid: 0000-0001-7258-9596
   wikidata: Q57023310
+- affiliation:
+    name: European Bioinformatics Institute, Cambridge
+    wikidata: Q695267
+  country: United Kingdom
+  github: souzadevinicius
+  groups:
+  - technical
+  name: Vinícius de Souza
+  orcid: 0000-0003-3961-0247
```

Comment:
@souzadevinicius you will have to add a wikidata identifier. Can you add one to wikidata and add it here?

Reply:
Done